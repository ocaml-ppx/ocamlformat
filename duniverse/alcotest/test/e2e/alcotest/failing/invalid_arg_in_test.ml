let () =
  let open Alcotest in
  run __FILE__
    [
      ( "test-a",
        [
          test_case "Passing" `Quick (fun () -> ());
          test_case "Failing" `Quick (fun () -> invalid_arg "Failing test");
        ] );
      ("test-b", [ test_case "Another pass" `Quick (fun () -> ()) ]);
    ]
