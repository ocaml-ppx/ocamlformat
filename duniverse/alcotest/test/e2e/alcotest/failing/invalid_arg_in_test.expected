Testing `test/e2e/alcotest/failing/invalid_arg_in_test.ml'.
This run has ID `<uuid>'.

  [OK]          test-a          0   Passing.
> [FAIL]        test-a          1   Failing.
  [OK]          test-b          0   Another pass.

┌──────────────────────────────────────────────────────────────────────────────┐
│ [FAIL]        test-a          1   Failing.                                   │
└──────────────────────────────────────────────────────────────────────────────┘
[invalid] Failing test

Logs saved to `<build-context>/_build/_tests/<test-dir>/test-a.001.output'.
 ──────────────────────────────────────────────────────────────────────────────

Full test results in `<build-context>/_build/_tests/<test-dir>'.
1 failure! in <test-duration>s. 3 tests run.
